lib/Conversion/CMakeLists.txt

```diff
@@ -30,4 +30,5 @@ add_subdirectory(CombToCGGI)
 add_subdirectory(MemrefToArith)
 add_subdirectory(PolynomialToStandard)
 add_subdirectory(SecretToBGV)
+add_subdirectory(SecretToCKKS)
```

Comment:
now I'm out of my cottage industry job to supply these files 😆

Reply:
To be fair I haven't tested it out yet! I'll finally try the local cmake build myself with this one haha